gui_constants.py

```diff
@@ -30,6 +30,8 @@ class GuiLabel:
     TELESCOPE_ERROR = "Errore"
     TELESCOPE_TRACKING_ON = "Track On"
     TELESCOPE_TRACKING_OFF = "Track Off"
+    TELESCOPE_SLEWING_ON = "Slewing"
+    TELESCOPE_SLEWING_OFF = "NO Slew"
```

Comment:
I rather prefer "Slewing on" and "Slewing off" as labels

Reply:
Too long terms for the label, then the font needs to be reduced